frontend/public/components/utils/simple-tab-nav.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import * as _ from 'lodash-es';
+
+class SimpleTab extends React.PureComponent<SimpleTabProps> {
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(e) {
+    e.preventDefault();
+    this.props.onClick(this.props.title);
+  }
+
+  render() {
+    const {active, title} = this.props;
+    const className = classNames('co-m-horizontal-nav__menu-item', {'co-m-horizontal-nav-item--active': active});
+    return <li className={className}>
+      <a href="#" onClick={this.onClick}>{title}</a>
```

Comment:
This would be better as a button with `btn btn-link` classes for accessibility, although I'm guessing that doesn't work with the existing tab styles.

Reply:
Probably not, but we could make it work.